packer/centos8-katello-nightly-stable.json

```diff
@@ -27,7 +26,7 @@
                 [ "-smp", "4" ]
             ],
             "boot_command": [
-              "<tab> text ks=http://{{ .HTTPIP }}:{{ .HTTPPort }}/ks.cfg<enter><wait>"
+              "<tab> text ks=http://{{ .HTTPIP }}:{{ .HTTPPort }}/ks8.cfg<enter><wait>"
```

Comment:
Where does `ks8.cfg` come from? I don't see the file being renamed.

Reply:
That was part of my testing, when we were doing both el 7/8, will fix that up and this pr shortly. Now we only need ks.cfg since we are just dealing with el8 only